src/components/HelpDrawer/HelpContent/SpectralProfilerSettingsFittingHelpComponent.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+export class SpectralProfilerSettingsFittingHelpComponent extends React.Component {
+    public render() {
+        return (
+            <div>
+                <h3>Profile Fitting</h3>
+                <p>With the fitting tab, users can fit the observed spectrum with a model profile. Please ensure that there is only one spectral profile in the plot. Fitting multiple observed spectra is not supported. The supported profile functions are:</p>
+                <ul>
+                    <li>Gaussian</li>
+                    <li>Lorentzian</li>
+                </ul>
+                <p>In addtion, an optional continuum profile can be added in the fit, including a constant profile and a linear profile.</p>
+                <h3>Set up initial solution for the fitter</h3>
+                <p>In order to make the profiler fitting engine work properly, a good set of initial solution of the free parameters need to be supplied. CARTA includes an <em>experimental</em> algorithm to analyze the input profile and makes an educated guess of the initial solution. This can be archieved by clicking the &quot;auto detect&quot; button. If there is a prominent continuum emission, please enable the &quot;w/cont.&quot; toggle before clicking the &quot;auto detect&quot; button.</p>     
```

Comment:
In order to work correctly, the profiler fitting engine must be supplied with a good set of initial solutions of the free parameters. CARTA includes an experimental algorithm which analyzes the input profile and makes an educated guess of the initial solution. To use it, click the "auto-detect" button. If there is...